src/libraries/Microsoft.PowerFx.Json/FormulaValueJSON.cs

```diff
@@ -22,14 +22,14 @@ public class FormulaValueJsonSerializerSettings
     public class FormulaValueJSON
     {
         /// <summary>
-        /// Convenience method to create a value from a json representation. 
+        /// Convenience method to create a value from a json representation.
         /// </summary>
-        public static FormulaValue FromJson(string jsonString, FormulaType formulaType = null, bool numberIsFloat = false)
+        public static FormulaValue FromJson(string jsonString, FormulaType formulaType = null, bool numberIsFloat = false, string path = null)
```

Comment:
> string path = null [](http://example.com/codeflow?start=122&length=19)

This is a breaking change. 
Why isn't the Pr pipeline flagging it.

Reply:
Investigating

Reply:
Resolved